src/hex.cpp

```diff
@@ -73,28 +73,31 @@ namespace libtorrent
 	extern char const hex_chars[];
 
 	char const hex_chars[] = "0123456789abcdef";
+	static void to_hex(char const* in, int const len, char* out)
```

Comment:
``in`` and ``out`` should probably be ``span<char const>`` and ``span<char>`` respectively here